packages/pods-core/src/builtin/__tests__/MarkdownPod.spec.ts

```diff
@@ -78,4 +78,129 @@ describe("MarkdownPublishPod", () => {
       }
     );
   });
+
+  test("schema with template", async () => {
```

Comment:
`schemaPreset` was to test that template schemas worked. I would use these tests more to test special cases for markdown (eg. what happens when we publish an empty note? a note with a wiki link? a note with a relative link? etc)

Reply:
Will think about some edge cases then 👍